src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -483,6 +486,9 @@ def update_monitoring_schedule(
                 self._normalize_endpoint_input(endpoint_input=endpoint_input)._to_request_dict()
             ]
 
+        elif batch_transform_input is not None:
```

Comment:
Can we also add a check so that if both `batch_transform_input` and `endpoint_input` are specified we raise an error?

Reply:
added the check